src/Boo.Lang.Compiler/Boo.Lang.Compiler.csproj

```diff
@@ -15,7 +15,8 @@
     <AllowUnsafeBlocks>False</AllowUnsafeBlocks>
     <NoStdLib>False</NoStdLib>
     <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>

```

Comment:
In the writeup, you say collectible assemblies were introduced in Framework version 4.0, which MSDN agrees with, but here (and elsewhere in the project) you set the minimum version to 4.5.  Why?